### PR TITLE
Add Header template

### PR DESCRIPTION
**Pull request by miina**
Adds `header.php` template, moves unnecessary things out of `search.php` and `index.php`.

### CODE REVIEW

**Comment by miina** [comment]
@kienstra Added header.php template, this should make it possible for running the site with AMP plugin active. Whenever you have time, would be great to review.

Note that the search.php and footer.php are still mostly placeholders.

**Comment by postphotos** [comment]
fixes #43.

File: footer.php [hunk 1]
```diff
@@ -9,73 +9,75 @@
...
-                                </button>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <!--/ Right column -->
+			<!-- Right column -->
+			<div class="travel-newsletter-signup">
+				<header class="max-width-3 mx-auto px1 md-px2">
+					<h4 class="travel-footer-right-column-heading travel-spacing-none h2 mb3 blue">Want travel deals<br>straight to your inbox?</h4>
```
Comment (kienstra) [inline footer.php:26]: This untranslated text is probably alright for now, as this template is a placeholder.

File: header.php [hunk 2]
```diff
@@ -15,31 +14,11 @@
 </head>
 <body <?php body_class(); ?>>
 
-<amp-state id="search_query">
```
Comment (kienstra) [inline header.php:18]: Thanks for moving these into `amp_travel_states()`.

File: index.php [hunk 3]
```diff
@@ -10,66 +10,8 @@
 // Ignore the issues of this file since that's just a copied HTML placeholder and contains some issues (e.g. including the scripts directly.)
 
 // @codingStandardsIgnoreFile
+get_header();
 ?>
-<!doctype html>
```
Comment (kienstra) [inline index.php:14]: Thanks for abstracting this into `header.php`.

**Review by kienstra — APPROVED** [review]
**Approved**
*Might Return For A Brief "Retro" Review*

@miina,
Thanks a lot for this pull request. It really helps to have the homepage 'Angles' appearing.

I'm going to merge this now, so this can be part of the screencast. I'll probably do a slightly more thorough "retro" review.

homepage-angles

**Pull request merged**